pom.xml

```diff
@@ -51,7 +51,7 @@
 	</modules>
 	
 	<properties>
-		<openmrsPlatformVersion>2.0.0</openmrsPlatformVersion>
+		<openmrsPlatformVersion>2.2.0</openmrsPlatformVersion>
```

Comment:
@rubailly  Any reason why you are not updating it to platform_version 2.3.2

Reply:
hi @rubailly ,did you try building this locally before pushing the changes ?